book/content/benchmarking/_from_duckdb.qmd

```diff
@@ -43,29 +43,31 @@ duckdb_arrow_dplyr <- function(variables) {
       min_colNum = min(colNum, na.rm = TRUE),
       mean_colNum = mean(colNum, na.rm = TRUE),
       max_colNum = max(colNum, na.rm = TRUE)
-    )
-  
+    ) |>
+    collect()
   
+  DBI::dbDisconnect(con, shutdown=TRUE)
+
   return(result)
   
 }
 tic()
-duckdb_arrow_dplyr()
+duckdb_dplyr()
 toc()
 ```
 
-## arrow/SQL/DuckDB
+## SQL/DuckDB
 
 ```{r}
-#| label: duckdb-arrow-sql-benchmarking
+#| label: duckdb-sql-benchmarking
```

Comment:
Perhaps duplicate label names?

Reply:
Thanks a lot @eitsupi  I will add in #14 a section on the complementarity of both syntaxes (arrow and dbplyr) using a more complex example